### PR TITLE
[JENKINS-54907] Prepare Docker image for JDK 11

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-54907](https://issues.jenkins-ci.org/browse/JENKINS-54907)

Specify the proper JAVA_OPTS in the Docker image so that JFR works with JDK 11.

Again based in #41

### CODE REVIEW

File: plugins.txt [hunk 1]
```diff
@@ -1 +1,4 @@
 pipeline-model-definition:latest
+workflow-support:3.0
```
Comment (varyvol) [inline plugins.txt:2]: Necessary plugins to execute [the cwp demo](https://github.com/jenkinsci/jenkinsfile-runner/tree/master/demo/cwp) in JDK 11.

**Review by oleg-nenashev — APPROVED** [review]
Should it be just a "jdk11" branch so that we configure DockerHub for it?

**Comment by varyvol** [comment]
@oleg-nenashev no strong opinion. I did not use that one because it contained your changes.

**Review by fcojfernandez — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
OK, I have removed the `jdk11` branch

**Comment by varyvol** [comment]
@oleg-nenashev do you know if branches can be renamed in github? if so we could merge and then rename the branch.

**Comment by oleg-nenashev** [comment]
Yes, let's do so

**Comment by oleg-nenashev** [comment]
Created https://github.com/jenkinsci/jenkinsfile-runner/tree/jdk11

**Pull request merged**